documentation/working-with-testcafe-studio/recording-tests/test-actions/on-page-actions/README.md

```diff
@@ -122,42 +122,42 @@ Parameter | Type  | Description
 
 You can also specify additional parameters for the action. See [Basic Action Options](action-options.md#basic-action-options).
 
-> You can add this action to the test only in the Test Editor, you cannot record it.
+> You can add this action to the test only in the Test Editor.
 
 ## Select Text Area Content
 
-Selects text within a text area element.
+Select text within a text element area.
 
 Parameter | Type  | Description
 --------- | ----- | -------------
-*[Element Selector](action-parameters.md#element-selector)* | [Selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/) | Identifies the text area whose text should be selected.
+*[Element Selector](action-parameters.md#element-selector)* | [Selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/) | Identifies the text that should be selected.
```

Comment:
@dirk-pieterse 

No, identifies namelly the text area (DOM element on the web page)

Reply:
Text area? Do you mean text range?

Reply:
Identifies the text area that should be selected?